pipelines/build.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+  platform: ''
+
+steps:
+# Prepare per platform
+- ${{ if eq(parameters.platform, 'windows') }}:
+  - script: tzutil /s "W. Europe Standard Time"
+    displayName: 'Fix Windows timezone'
+
+- ${{ if eq(parameters.platform, 'darwin_arm') }}:
+  - bash: |
+      node -v && node -p "process.arch"
+      rm -rf dist
+    displayName: Use preinstalled node 18.17.1
+
+- ${{ if eq(parameters.platform, 'linux') }}:
+  - bash: |
+      sudo apt-get update
+      sudo apt-get install libudev-dev
+    displayName: 'Prepare linux'
+
+- ${{ if ne(parameters.platform, 'darwin_arm') }}:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: $(NODE_VERSION)
+    displayName: 'Install Node.js $(NODE_VERSION)'  
```

Comment:
Would be nice to move this whole group of steps into its own `prepare.yml`.

Reply:
Yeah, I was actually considering using that exact name as well